webapp/portlet/src/main/webapp/vue-apps/space-settings/components/SpaceSettings.vue

```diff
@@ -24,19 +25,21 @@ export default {
   }),
   created() {
     // add external components
-    const externalComponents = extensionRegistry.loadComponents('external-space').map(component => component.componentOptions.componentImpl);
+    const externalComponents = extensionRegistry.loadComponents('external-apps-space-settings').map(component => component.componentOptions.componentImpl);
     this.spaceExternalSettings.push(...externalComponents);
-
-    document.addEventListener('addSpaceSettingsExternalComponents', (event) => {
-      if (event && event.detail) {
-        this.spaceExternalSettings.push(event.detail.componentImpl);
-      }
-    });
-
     document.addEventListener('hideSettingsApps', () => this.displayed = true);
     document.addEventListener('showSettingsApps', () => this.displayed = true);
   },
   mounted() {
+    document.addEventListener('component-external-apps-space-settings-space-settings-updated', (event) => {
+      if (event && event.detail ) {
+        this.$spaceService.getSpaceApplications(this.spaceId).then(applications => {
```

Comment:
We should retrieve Space Applications only once IMO outside this event. This call will be used to retrieve the same space's applications for each **space settings extension**, isn't it ?